KsApi/models/Project.swift

```diff
@@ -25,6 +25,10 @@ public struct Project {
   public var urls: UrlsEnvelope
   public var video: Video?
 
+  public static func decode(id: Int) -> String {
```

Comment:
Can you add test for this function?

Reply:
I think from my comment [here](https://github.com/kickstarter/ios-oss/pull/827#discussion_r320845807) we actually don't need this either because the `GraphIdBridging` protocol takes care of it for us.

Reply:
@cdolm92 can we remove this?